base-images/build.sh

```diff
@@ -31,13 +31,15 @@ docker buildx build \
     --pull --push --platform linux/x86_64,linux/arm64 \
     -t ${DOCKER_HUB_ORG}/python:3.8-ubuntu20.04 \
     -f python3.8-ubuntu20.04.Dockerfile .
+
+# TODO(gaocegege): Support linux/arm64
 docker buildx build \
     --build-arg ENVD_VERSION=${ENVD_VERSION} \
     --build-arg ENVD_SSH_IMAGE=ghcr.io/tensorchord/envd-ssh-from-scratch \
     --build-arg HTTP_PROXY=${HTTP_PROXY} \
     --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
-    -t ${DOCKER_HUB_ORG}/r-base:4.2  \
-    --pull --push --platform linux/x86_64,linux/arm64 \
+    -t ${DOCKER_HUB_ORG}/r-base:4.2 \
+    --pull --push --platform linux/x86_64 \
```

Comment:
We should probably automate the release of images as part of the release process

Reply:
SGTM! I will open an issue to keep track.

Reply:
#545